tiger/Cargo.toml

```
@@ -0,0 +1,25 @@
+[package]
+name = "tiger"
+version = "0.1.2"
```

I'd suggest setting this to:

```suggestion
version = "0.0.0"
```

We can bump it on release.